model/standalone_driver/src/icon4py/model/standalone_driver/standalone_driver.py

```diff
@@ -0,0 +1,786 @@
+        ]
+        icon4py_signature += [
+            "*            -------                                                           //         *"
+        ]
+        icon4py_signature += [
+            "*                                                                             //          *"
+        ]
+        icon4py_signature += [
+            "*                                                                = = = = = = //           *"
+        ]
+
+        for _ in range(3):
+            icon4py_signature += empty_line
+        icon4py_signature += boundary_line
+        icon4py_signature = "\n".join(icon4py_signature)
+        self._logger.info(f"{icon4py_signature}")
+
+        self._logger.info("===== ICON4Py Driver Configuration =====")
+        self._logger.info(f"Experiment name        : {self.config.experiment_name}")
+        self._logger.info(f"Time step (dtime)      : {self.config.dtime.total_seconds()} s")
+        self._logger.info(f"Number of timesteps    : {self.n_time_steps}")
+        self._logger.info(f"Initial ndyn_substeps  : {self.config.ndyn_substeps}")
+        self._logger.info(f"Vertical CFL threshold : {self.config.vertical_cfl_threshold}")
+        self._logger.info(
+            f"Second-order divdamp   : {self.config.apply_extra_second_order_divdamp}"
+        )
+        self._logger.info(f"Statistics enabled     : {self.config.enable_statistics_output}")
+        self._logger.info("")
+
+        self._logger.info("==== Vertical Grid Parameters ====")
+        self._logger.info(self.solve_nonhydro._vertical_params)
+        self._logger.info(self._format_physics_constants())
+
+    def _make_timers(self) -> None:
+        self._timers: dict[str, Timer] = {}
+        self._timers["timer_solve_nh_first_step"] = Timer("Solve nh: first time step", dp=6)
+        self._timers["timer_solve_nh"] = Timer("Solve nh: after first time step", dp=6)
+        self._timers["timer_diffusion_first_step"] = Timer("Diffusion: first time step", dp=6)
+        self._timers["timer_diffusion"] = Timer("Diffusion: after first time step", dp=6)
+
+    def _make_logger(self) -> None:
+        """
+        Create a log file and two log formats for debug and other logging levels. When debug level is used for
+        a message, an ascii time stamp and the function name are appended to the beginning of the message.
+        """
+        self._logger = logging.getLogger("Icon4PyDriver")
+        file_handler = logging.FileHandler(
+            filename=str(self.config.output_path.joinpath(f"log_{self.config.experiment_name}")),
+        )
+        logging_formatter = _DerivedFormatter(
+            style="{",
+            default_fmt="{message}",
+            debug_fmt="{asctime} {funcName:<20} : {message}",
+        )
+        file_handler.setFormatter(logging_formatter)
+        file_handler.setLevel(logging.DEBUG)
+        self._logger.addHandler(file_handler)
+
+    def _initialize_timeloop_parameters(self) -> None:
```

Comment:
maybe as free function
```suggestion
def _initialize_timeloop_parameters(self, config) -> RuntimeValues:
    ...
    return RuntimeValues(....)
```

or 
```python
@dataclasses.dataclass
class RuntimeValues:
    ndyn_substeps_var: int
    ...
    config: dataclasses.InitVar[DriverConfig]
    def __init__(self, config) -> None:
        ... # all the stuff from _initialize_timeloop_parameters
```